source/snippets/install_notes.rst_

```diff
@@ -0,0 +1,10 @@
+.. note::
+
+   阅读此安装说明前请注意：
+
+   1. 我们假定用户已经根据《:doc:`seis101:computer/setup`\ 》中的内容安装了
+      C/C++ 和 Fortran 编译器 ``gcc``、``g++`` 和 ``gfortran``
+      以及常用命令行工具 ``make`` 和 ``git`` 等。
```

Comment:
gcc, g++, gfortran, make, git: these are the 5 commonly used commands. We may add more later.

Reply:
Yes~